.github/workflows/create-wheels.yaml

```diff
@@ -9,7 +9,7 @@ env:
   # set this so the falcon test uses the installed version and not the local one
   PYTHONNOUSERSITE: 1
   # comment TWINE_REPOSITORY_URL to use the real pypi. NOTE: change also the secret used in TWINE_PASSWORD
-  TWINE_REPOSITORY_URL: https://test.pypi.org/legacy/
+  # TWINE_REPOSITORY_URL: https://test.pypi.org/legacy/
```

Comment:
do we want to target the real pipy? (if we do we need to also change the secret in the configuration)

Reply:
will wait for kurt, but I can revert that without problems